From: Richard B Winters <rik@mmogp.com>
Date: Tue, 5 May 2015 16:04:56 -0400
Subject: Do not install upstream pcsd init script

 - Remove instruction to install the upstream pcsd init from 
   the Makefile, as we will install a maintainer provided init 
   script instead

Change-Id: I1af25537d60d59be8c92cfb481b104bd36e9df30
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index e0073e3..f3c412b 100644
--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,6 @@ ifeq ($(IS_SYSTEMCTL),true)
 	install -d ${DESTDIR}/lib/systemd/system
 	install -m 644 pcsd/pcsd.service ${DESTDIR}/lib/systemd/system/
 endif
-	install -m 755 -D pcsd/pcsd ${DESTDIR}/${initdir}/pcsd
 
 uninstall:
 	rm -f ${DESTDIR}${PREFIX}/sbin/pcs
